### DISCUSSION TITLE
Method to get values from a label

### DISCUSSION THREAD

**Discussion by Ezekiel-Webfox** [webfox/laravel-backed-enums#38] [Ideas]
Could we get a method to retrieve values from a given label.

Would be useful in situations where user is searching using the enum label, but we need to filter using the value.

**Comment by Ezekiel-Webfox** [comment]
In the meantime can filter on the labels array, but I think it may be a useful convenience method.

**Comment by hailwood** [comment]
@Ezekiel-Webfox how would you handle the case of there being multiple items that have the same label? Return the first found? Return an array if there are multiple found? Always return an array?

**Comment by Ezekiel-Webfox** [comment]
@hailwood I would say always return an array for consistency
